### PR TITLE
ADD TLS is not supported limit

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What problem does this PR solve? 
https://github.com/pingcap/tispark/issues/2266 TiSpark doesn't support TLS now, so add limit in readme.
by the way, we will support it in the future

### CODE REVIEW

**Comment by shiyuhang0** [comment]
PTAL @birdstorm @zhangyangyu

File: README.md [hunk 1]
```diff
@@ -278,7 +278,9 @@ For more details about the test, see [here](./core/src/test/Readme.md).
 
 5. The dependency `tispark-assembly` should not be packaged into `JAR of JARS` file (for example, build with spring-boot-maven-plugin), or you will get `ClassNotFoundException`. You can solve it by adding `spark-wrapper-spark-version` in your dependency or constructing another forms of jar file.
 
-6. Tispark don't support collations now. `new_collations_enabled_on_first_bootstrap` should not be set to `true`.
+6. TiSpark doesn't support collations now. `new_collations_enabled_on_first_bootstrap` should not be set to `true`.
+
+7. TiSpark doesn't support TLS now. You can't visit TiDB with TLS.
```
Comment (zhangyangyu) [inline README.md:283]: visit  -> connect

**Review by zhangyangyu — APPROVED** [review]

**Review by xuanyu66 — APPROVED** [review]

**Review by sunxiaoguang — APPROVED** [review]

**Pull request merged**